tools/ir-generator/irclass.cpp

```diff
@@ -530,7 +530,7 @@ int IrClass::generateConstructor(const ctor_args_t &arglist, const IrMethod *use
     }
 
     if (kind == NodeKind::Abstract) ctor->access = IrElement::Protected;
-    ctor->inImpl = true;
+    ctor->inImpl = false;
```

Comment:
This will have major effect on the size of the ir-generated.h header. Is this necessary?

Reply:
We have some deeply nested inheritance trees. Wouldn't this lead to a blowup if every constructor can be inlined?